### ISSUE TITLE
Update bronze sponsor listings on sponsor page

### ISSUE THREAD

**Issue by nswitzer** [DrupalCampColorado/dcco2018#49]
For sponsors in the bronze tier, the logo in the sponsor grid should link to their website, instead of a detail page. Please also add Rabbit Hole so we can redirect bronze sponsor detail pages to the top level sponsor page.

**Comment by jnettik** [comment]
This is done.

**Comment by nswitzer** [comment]
Thanks @jnettik - I tested this out on the dev site and it looks like the bronze sponsor logos have the link removed completely. Could you update this so that the logo links to the sponsor website (the value in field_sponsor_url) if there is a link in that field? Could you also please add rabbit hole so that we can redirect the sponsor detail pages for bronze sponsors back to /sponsor?

Test page: https://full2018.drupalcampcolorado.org.develop.us2.compact.amazee.io/sponsor

**Comment by jnettik** [comment]
Yeah, not sure how I missed that...

Bronze sponsors should get linked to their websites now. I've also added a redirect to `/sponsor` for the bronze tier pages. Unless you're a logged in admin, those users can still view the full page when logged in. I just wrote a custom redirect over adding the Rabbit Hole module, that way I could have additional control over when the redirect happened. If you have strong opinions on using Rabbit Hole, let me know and I can play around with it some more.
